docs/02-connect/01-get_a_wallet.md

```diff
@@ -4,14 +4,398 @@ title: Get a Wallet
+</p>
+
+### Create an Account
+To begin, create an account for your wallet. Make sure that your password meets the following criteria:
+
+* Your password must contain at least 16-characters.
+* No common passwords are accepted, such as password, 12345, etc.
+* Your password cannot be saved to a keychain, therefore keep your password in a safe place.
+
+Perform the following steps:
+
+1. Enter a **password**. 
+2. Enter your password again to **confirm**.
+3. Click **Terms of Use**. After reading the terms and conditions, click the **checkbox** to agree on the terms.
+4. Click **Create**.
+
+
+### Create a New Wallet
+
+1. To create the new wallet, click **Create Wallet**.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-getstarted.png" alt="Cobalt 2-sidechains" width="300" height="200" />
+</p>
+
+2. The **Choose a wallet** page appears. Click **Horizen EON** or **TokenMint** to create a wallet on a supported sidechain. 
+
+<p>
+<img src="/img/docs/get-started/cobalteon-choose-wallet.png" alt="Cobalt 2-sidechains" width="300" height="200" />
+</p>
+
+3. The **Create wallet for EON Testnet** page appears. Enter your wallet’s name. Click **Next**. 
+
+
+<p>
+<img src="/img/docs/get-started/cobalteon-wallet-name.png" alt="Cobalt WalletName" width="300" height="200" />
+</p>
+
+<p>
+<img src="/img/docs/get-started/cobalteon-wallet-name.png" alt="Cobalt Wallet Name" width="300" height="200" />
+</p>
+
+4. The **Seed Phrase** page appears. Write down your seed phrase. Click **Next**.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-FriedSard-seed.png" alt="Cobalt Wallet Name" width="300" height="200" />
+</p>
+
+5. The **Confirm seed phrase** page appears. Select the word in the correct order to confirm your seed phrase. 
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-FSseed-confirm.png" alt="Cobalt SeedConfirm" width="300" height="200" />
+</p>
+
+6. Click **Confirm**. Cobalt displays your wallet name.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-wallet-complete.png" alt="Cobalt Wallet complete" width="300" height="200" />
+</p>
+
+### Import a Wallet
+
+To restore a previously created wallet, use the **Import Wallet** option. Before you restore a wallet, you need the wallet’s name, password, and seed phrase. 
+
+After installing Cobalt, perform the following steps to import a wallet:
+
+1. In the **Get Started** page, click **Import Wallet**.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-import-wallet.png" alt="Cobalt Import Wallet" width="300" height="200" />
+</p>
+
+2. The **Choose a wallet** page appears. Select either **Horizen EON** or **TokenMint** for your public sidechain. 
+
+<p>
+<img src="/img/docs/get-started/cobalteon-importwallet-choose.png" alt="Cobalt Choose Sidechain" width="300" height="200" />
+</p>
+
+3. The **Importing wallet for TokenMint Chain** page appears. Enter the wallet name and password. Click **Next**. 

```

Comment:
Let's show the flow when clicking "Horizen EON" bc this is EON documentation.
In that case, it will be a 12 word seed phrase
Please also update the screenshot

Reply:
Updated the flow to show Import Wallet for Horizen Gobi Testnet with new screenshots

Reply:
@Ramone-HL, can you also update the next step (step 4) where it says "Enter your wallet’s 24-words seed phrase." to "Enter your wallet’s 12-words seed phrase." to reflect that of Horizen EON's requirement?
 
<img width="747" alt="Screenshot 2023-04-26 at 1 17 42 PM" src="https://user-images.githubusercontent.com/5024180/234653029-a77c6b2d-77e7-43c3-8431-ce49db7ffb7c.png">